gen3/docs/gen3-resources/glossary.md

```diff
@@ -102,3 +102,5 @@ Gen3 workspaces are secure data analysis environments in the cloud that can acce
 [SDK docs]: /gen3-resources/user-guide/search/#exporting-structured-data-programmatically
 [PFB GitHub]: https://github.com/uc-cdis/pypfb
 [PFB Pub]: https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010944
+[here]: /gen3-resources/user-guide/analyze-data/
```

Comment:
I think this reference should be "workspace use" not "here". And in the next line, the reference should be "workspace set up" and needs a link

Reply:
Oh, good catch.  I fixed the [here] link, but the other one I think still needs to be determined.  Do you talk about this anywhere in your helm docs?  I just removed that sentence for now and we can revisit later.  How does that sound?

Reply:
I definitely do talk about how to configure things for workspaces, so I think that works for now